### ISSUE TITLE
one subdomain for use central

### ISSUE THREAD

**Issue by webactor724** [stancl/tenancy#1150]
#### Description

I want the domain to be on WordPress, for example, site.com is on WordPress and with a specific IP that is completely separate from the tenancy system, but I want to have a subdomain named app.site.com where I can join and create a tenant. But

But I have an idea. I want to define 2 domains, one in the central domains
app.site.com
 site.com
My goal is to run the subdomains that I consider for tenants in the main subdomain, for example
vahid.site.com
But I have a problem

When I want to enter a tenant through the impresonate method, the address is like this

app.site.com/impersonate/xxxxxxxxx
And I get an error that app.site.com does not include a subdomain.

In fact, I am not allowed to execute a request as centeal domain, which is a subdomain

#### Why this should be added
 The purpose of separating the introduction section of the project in the main domain and an app subdomain for platform management and other subdomains for tenants

**Comment by kevillalba** [comment]
Yes, what you want to achieve is possible using DNS configuration. For example, in Cloudflare you can configure:

site.com → WordPress (separate IP).
app.site.com → Laravel application (central domain).
*.site.com → Wildcard DNS record pointing to the same Laravel server for all tenants (tenant1.site.com, tenant2.site.com, etc.).

There is no need to create an additional subdomain for the central application. app.site.com can be your central domain, while *.site.com serves all tenant subdomains.

If app.site.com/impersonate/... is failing because it says there is no subdomain, the issue is not with the DNS configuration but with the ArchTechX Tenancy configuration. app.site.com should be registered as a central domain, and the impersonate route should be handled in the central domain context, not as a tenant.
